pkg/provider/aws/client.go

```diff
@@ -44,7 +44,10 @@ type AwsClientInput struct {
 	Region          string
 }
 
-const ProxyConfigKey = "aws_proxy"
+const (
+	ProxyConfigKey = "aws_proxy"
+	NoProxyFlag    = "no-aws-proxy"
```

Comment:
Suggestion for consistency for other skip flags.
```suggestion
	NoProxyFlag    = "skip-aws-proxy-check"
```